addressing/README.md

```diff
@@ -147,6 +147,33 @@ within it. Using our example above, decapsulating either `/tcp/1234/ws` _or_
 unsurprising if you consider the utility of the `/ip4/7.7.7.7/ws` address that
 would result from simply removing the `tcp` component.
 
+### The security multiaddr
```

Comment:
```suggestion
### The multiaddr security component
```

While I know that each multiaddr component (e.g. `/ip6/::1` or `/tcp/1234`) in itself is a valid multiaddr, I don't think there is much value in a single component `/tls` multiaddr. Thus instead of speaking of _the security multiaddr_, I would refer to something along the lines of the _security multiaddr component_.

Reply:
Good point.